src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/updateCVEDescription.hpp

```diff
@@ -0,0 +1,138 @@
+/*
+ * Wazuh Vulnerability scanner - Database Feed Manager
+ * Copyright (C) 2015, Wazuh Inc.
+ * Oct 3, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef _UPDATE_CVE_DESCRIPTION_HPP
+#define _UPDATE_CVE_DESCRIPTION_HPP
+
+#include "chainOfResponsability.hpp"
+#include "cve5_generated.h"
+#include "eventContext.hpp"
+#include "flatbuffers/flatbuffers.h"
+#include "flatbuffers/verifier.h"
+#include "rocksDBWrapper.hpp"
+#include "vulnerabilityDescription_generated.h"
+
+constexpr auto DESCRIPTION_DATABASE_PATH {"queue/vd/descriptions"};
+constexpr auto CVE5_DATABASE_PATH {"queue/vd/cve5"};
+
+/**
+ * @brief UpdateCVEDescription class.
+ *
+ */
+class UpdateCVEDescription final : public AbstractHandler<std::shared_ptr<EventContext>>
+{
+public:
+    /**
+     * @brief Reads CVE5 database, creates a vulnerability description flatbuffer and stores it in a specific RocksDB
+     * database.
+     *
+     * @param resource CVE ID.
+     * @param buf Flatbuffer pointer.
+     * @param size Flatbuffer size.
+     */
+    static void storeVulnerabilityDescription(const std::string& resource, uint8_t* buf, const size_t& size)
+    {
+        flatbuffers::Verifier verifier(buf, size);
```

Comment:
remove this verification, probably this was verified in previous steps.

Reply:
Changed